### PR TITLE
bump buck to 2020.05.14.01, NDK to 20.1.5948944

### PR DESCRIPTION
**Pull request by dulmandakh**
# Summary

Bump buck to 2020.05.14.01, NDK to 20.1.5948944

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅❌     |
| Android |    ✅❌     |

## Checklist

- [ ] I have tested this on a device and a simulator
- [ ] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [ ] I updated the typed files (TS and Flow)
- [ ] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

**Comment by gengjiawen** [comment]
CI looks failed when installing buck

**Comment by gengjiawen** [comment]
Is this caused by new buck ?

```console
+ buck fetch ReactAndroid/src/test/java/com/facebook/react/modules
Not using buckd because watchman isn't installed.
WARNING: Output when parsing /pwd/ReactAndroid/src/test/java/com/facebook/react/modules/BUCK:
| Traceback (most recent call last):
|   File "/pwd/buck-out/tmp/buck_run.jr5wbb1j/buck_python_program4424192714511431925/__main__.py", line 13, in <module>
|     from buck_parser import buck
|   File "/pwd/.buckd/resources/2020.05.14.01/buck_server/buck_parser/buck.py", line 23, in <module>
| ImportError: No module named json
PARSING BUCK FILES: FINISHED IN 1.8s
Buck wasn't able to parse /pwd/ReactAndroid/src/test/java/com/facebook/react/modules/BUCK:
```

File: Dockerfile [hunk 1]
```diff
@@ -5,11 +5,11 @@ LABEL Description="This image provides a base Android development environment fo
...
 
 # set default build arguments
-ARG SDK_VERSION=sdk-tools-linux-4333796.zip
+ARG SDK_VERSION=commandlinetools-linux-6514223_latest.zip
 ARG ANDROID_BUILD_VERSION=29
 ARG ANDROID_TOOLS_VERSION=29.0.3
-ARG BUCK_VERSION=2019.10.17.01
-ARG NDK_VERSION=20.0.5594570
+ARG BUCK_VERSION=2020.05.14.01
+ARG NDK_VERSION=20.1.5948944
```
Comment (gengjiawen) [inline Dockerfile:12]: Will `21.2.6472646` ready for react native ?

File: Dockerfile [hunk 2]
```diff
@@ -19,7 +19,7 @@ ENV ANDROID_HOME=/opt/android
 ENV ANDROID_SDK_HOME=${ANDROID_HOME}
 ENV ANDROID_NDK=${ANDROID_HOME}/ndk/$NDK_VERSION
```
Comment (gengjiawen) [inline Dockerfile:20]: Thought on adding `ANDROID_NDK_HOME` too ?

File: Dockerfile (hunk 1 above)
Comment (dulmandakh) [reply Dockerfile:12]: RN is failing to compile with NDK 21

**Comment by dulmandakh** [comment]
@gengjiawen please merge and publish

**Comment by gengjiawen** [comment]
Thanks, merged.

**Comment by gengjiawen** [comment]
Publish is auto, you can check here:  https://circleci.com/gh/react-native-community/docker-android/153?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link.

**Comment by dulmandakh** [comment]
publish failed

**Comment by gengjiawen** [comment]
> publish failed

I retried, looks success.

**Pull request merged**